index.js

```diff
@@ -49,6 +50,11 @@ if (options.cacheTime !== 0) {
 /*
 Process query parameters
  */
+
+function isUrl (input) {
```

Comment:
because of the way I implemented, the following will actually work in the original version
```PowerShell
$ site-validator alheimsins.net
```
but this will fail the ```isUrl``` test because it does not have the protocol.

I wonder if we should change the approach to ```isFile(input)``` instead, where we check if a file exists, if not then we convert it to a url using the normalizer and try to parse that.

Note: at least on Windows, ```alheimsins.net``` is a valid folder/file name

Reply:
nice catch

I'm on Mac. But isFile is fine by me. I'll fix it

Reply:
OK! I'll wait for an updated version!

Reply:
thought I fixed it, needs more work

Reply:
Apologies, I'll close this and review the change there.